ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/SLES.yml

```diff
@@ -47,5 +47,6 @@ Test_Tool_Packages:
   - xorg-x11
   - xorg-x11-server
   - xorg-x11-server-extra
+  - glibc-locale
```

Comment:
Can you add a comment as to why this was added

Reply:
Thank you for your support, @karianna . I've updated the line.